src/Primitives/src/StringSegment.cs

```diff
@@ -656,9 +656,9 @@ public static bool IsNullOrEmpty(StringSegment value)
         }
 
         /// <summary>
-        /// Returns the <see cref="string"/> represented by this <see cref="StringSegment"/> or <code>String.Empty</code> if the <see cref="StringSegment"/> does not contain a value.
+        /// Returns the <see cref="string"/> represented by this <see cref="StringSegment"/> or <c>String.Empty</c> if the <see cref="StringSegment"/> does not contain a value.
```

Comment:
This should've been a cref to the actual String.Empty field:

```suggestion
        /// Returns the <see cref="string"/> represented by this <see cref="StringSegment"/> or <see cref="String.Empty" /> if the <see cref="StringSegment"/> does not contain a value.
```